### ISSUE TITLE
padding_mode/borderMode for RandomAffine

### ISSUE THREAD

**Issue by qxcv** [kornia/kornia#635]
## 🚀 Feature

It would be helpful to expose `padding_mode` (OpenCV's `borderMode`) in the `RandomAffine` augmentation.

## Motivation

Controlling padding mode in augmentations may or may not make it easier to train networks. In any case, this is a useful hyperparameter to expose. A feature along these lines was mentioned in https://github.com/kornia/kornia/issues/476#issuecomment-615838590, but hasn't been done yet, so I'm volunteering. 

## Pitch

This is a simple change that would can work in roughly the same way that the `resample` keyword argument already works.

## Alternatives

The only alternative I can think of is pre-padding images before passing them to `RandomAffine`, then cropping them afterwards. This is obviously inefficient.

## Additional context

Let me know if there are other kwargs that I should be exposing when I make this change, or if I should be doing the same thing to other augmentation classes.

**Comment by shijianjian** [comment]
Great. That is one thing we missed previously!

In TorchVision, they used [fillcolor](https://pytorch.org/docs/stable/torchvision/transforms.html#torchvision.transforms.RandomAffine) than padding_mode. If we are going to have the padding_mode stuff, I think we may have both padding_mode and fillcolor.

What do you think @edgarriba?
